gr-analog/python/analog/qa_agc.py

```diff
@@ -492,6 +492,39 @@ def test_006_003_agc3_setter_errors(self):
             with self.assertRaises(ValueError):
                 troublemaker(-3.0)
 
+    def test_006_004_agc3(self, stride=1):
```

Comment:
Can you please give this either a more descriptive name, or add a docstring to explain what this does?